src/firefly/js/templates/lightcurve/LcResult.jsx

```diff
@@ -2,56 +2,97 @@
  * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
  */
 
-import React, {Component} from 'react';
+import React, {Component, PropTypes} from 'react';
 import sCompare from 'react-addons-shallow-compare';
-import {pick} from 'lodash';
+import {pick, get, isEmpty, set} from 'lodash';
 import SplitPane from 'react-split-pane';
-
-
 import {flux} from '../../Firefly.js';
 import {LO_VIEW, getLayouInfo} from '../../core/LayoutCntlr.js';
 import {TablesContainer} from '../../tables/ui/TablesContainer.jsx';
 import {ChartsContainer} from '../../charts/ui/ChartsContainer.jsx';
 import {VisToolbar} from '../../visualize/ui/VisToolbar.jsx';
 import {LcImageViewerContainer} from './LcImageViewerContainer.jsx';
 import {createContentWrapper} from '../../ui/panel/DockLayoutPanel.jsx';
-import {LC} from './LcManager.js';
-import {Tabs, Tab} from '../../ui/panel/TabPanel.jsx';
+import {LC, updateLayoutDisplay} from './LcManager.js';
+import {getTypeData, ReadOnlyText} from './LcPeriod.jsx';
+import {FieldGroup} from '../../ui/FieldGroup.jsx';
+import FieldGroupUtils from '../../fieldGroup/FieldGroupUtils';
+import {SuggestBoxInputField} from '../../ui/SuggestBoxInputField.jsx';
 import {ValidationField} from '../../ui/ValidationField.jsx';
-import {UploadPanel} from './LcViewer.jsx';
 import {LcImageToolbar} from './LcImageToolbar.jsx';
-import {LcPFOptionsPanel} from './LcPhaseFoldingPanel.jsx';
-import {LcPeriodFindingPanel} from './PeriodogramOptionsPanel.jsx';
 import {DownloadButton, DownloadOptionPanel} from '../../ui/DownloadDialog.jsx';
+const resultItems = ['title', 'mode', 'showTables', 'showImages', 'showXyPlots', 'searchDesc', 'images',
+                     LC.MISSION_DATA, LC.GENERAL_DATA, 'periodState'];
+const labelWidth = 100;
+
+const cTimeSeriesKeyDef = {
+    time: {fkey: LC.META_TIME_CNAME, label: 'Time Column'},
+    flux: {fkey: LC.META_FLUX_CNAME, label: 'Flux Column'},
+    timecols: {fkey: LC.META_TIME_NAMES, label: ''},
+    fluxcols: {fkey: LC.META_FLUX_NAMES, label: ''},
+    cutoutsize: {fkey: 'cutoutSize', label: 'Cutout Size (deg)'},
+    errorcolumn: {fkey: 'errorColumn', label: 'Error Column'}
+};
+
+const defValues = {
+    [cTimeSeriesKeyDef.time.fkey]: Object.assign(getTypeData(cTimeSeriesKeyDef.time.fkey, '',
+                                                'time column name',
+                                                `${cTimeSeriesKeyDef.time.label}:`, labelWidth),
+                                                {validator: null}),
```

Comment:
The value of this object look similar.  Could that be a in function to make the code a little more readable?